cocos/ui/scroll-view.ts

```diff
@@ -1240,13 +1276,12 @@ export class ScrollView extends ViewGroup {
 
         const originalMoveLength = deltaMove.length();
         let factor = targetDelta.length() / originalMoveLength;
-        targetDelta.add(deltaMove);
 
         if (this.brake > 0 && factor > 7) {
             factor = Math.sqrt(factor);
-            const clonedDeltaMove = deltaMove.clone();
-            clonedDeltaMove.multiplyScalar(factor);
-            targetDelta.set(clonedDeltaMove);
+            targetDelta.set(deltaMove);
+            targetDelta.multiplyScalar(factor + 1);
```

Comment:
```
const clonedDeltaMove = deltaMove.clone();    // clonedDeltaMove = deltaMove
clonedDeltaMove.multiplyScalar(factor);           // clonedDeltaMove = clonedDeltaMove * factor = deltaMove * factor
targetDelta.set(clonedDeltaMove);                    // targetDelta = clonedDeltaMove = deltaMove * factor
targetDelta.add(deltaMove);                               // targetDelta = targetDelta + deltaMove = deltaMove * factor + deltaMove = deltaMove * (factor + 1)
```

Reply:
Why does the factor need to be increased?

Reply:
Please refer to the comment above.